antiplag/constants.py

```diff
@@ -1,4 +1,5 @@
 CONTENT_TYPE_FILE = "multipart/form-data"
-CONTENT_TYPE_TEXT = "text/plain"
+CONTENT_TYPE_TEXT = "application/json"
```

Comment:
According to [media-types](https://www.iana.org/assignments/media-types/media-types.xhtml), `application/json` is not `CONTENT_TYPE_TEXT`. 

Please change name accordingly or explain the reasoning behind this.

Reply:
This is what we agreed on that will be sent from BE not plain text anymore, discussed with @MCFreddie777 Do you want to chime in?
But overall it won't be plain text anymore because we attach some data, e.g. in my case we attach emails so we're changing the type to json. The raw_text is still in the request.

Reply:
When I was coding this - it does not mean content_type_<type of content file> but content_typ_<type of our logic>.

Therefore  `CONTENT_TYPE_TEXT` stands for "Get me value of content type which is used during text upload."
And analogically - `CONTENT_TYPE_FILE` means `multipart/form-data` is content type used during file upload.

Reply:
Well but then it's not `CONTENT_TYPE_TEXT` anymore. 

Please update the constant's name.

Reply:
Yes makes sense, changing to CONTENT_TYPE_JSON